.bazelrc

```diff
@@ -9,9 +20,19 @@ build:rustfmt --output_groups=+rustfmt_checks
 build:clippy --aspects=//rust:defs.bzl%rust_clippy_aspect
 build:clippy --output_groups=+clippy_checks
 
-# https://bazel.build/docs/windows#symlink
-startup --windows_enable_symlinks
+###############################################################################
+## Incompatibility flags
+###############################################################################
+
+# https://github.com/bazelbuild/bazel/issues/8195
+build --incompatible_disallow_empty_glob=true
```

Comment:
I'm nervous that changes to `crate_universe` could trigger a regression here so thought I'd always enable it.